app/src/js/components/Section/section.js

```diff
@@ -15,7 +15,7 @@ export const section = ({
 }) => {
   const borderMod = border ? '--border' : '';
   const sizeMod = `--${size}`;
-  const id = sectionId || false;
+  const id = sectionId || 'false';
```

Comment:
Fix error in the JavaScript Console:
react_devtools_backend.js:2574 Warning: Received `false` for a non-boolean attribute `id`.